docs/reference/install.md

```diff
@@ -64,203 +44,20 @@ installed, then you can use the `master` channel.
 To install the Helm chart from master, you will need to pass the specific
 version returned by the `search` command:
 
-Install with Helm 2:
-
-```console
-helm repo add crossplane-master https://charts.crossplane.io/master/
-helm search crossplane-master
-helm install --name crossplane --namespace crossplane-system crossplane-master/crossplane --version <version>
-```
-
-For example:
-
-```console
-helm install --name crossplane --namespace crossplane-system crossplane-master/crossplane --version 0.0.0-249.637ccf9
-```
-
-Install with Helm 3:
-
-If your Kubernetes version is lower than 1.15 and you'd like to install
-Crossplane via Helm 3, you'll need Helm v3.1.0+.
-
 ```console
 kubectl create namespace crossplane-system
 helm repo add crossplane-master https://charts.crossplane.io/master/
 helm search repo crossplane-master --devel
 
-# Kubernetes 1.15 and newer versions
 helm install crossplane --namespace crossplane-system crossplane-master/crossplane --version <version> --devel
-
-# Kubernetes 1.14 and older versions
-helm install crossplane --namespace crossplane-system crossplane-master/crossplane --version <version> --devel --disable-openapi-validation
-```
-
-## Installing Infrastructure Providers
```

Comment:
Went ahead and removed this section and will replace it as part of https://github.com/crossplane/crossplane/issues/1726